keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -849,15 +850,22 @@ in back-to-back releases.
 
 #### Beta
 
-- e2e tests
+- e2e tests for coordinated leader election on various scenarios
+  + single leasecandidate
+  + multiple leasecandidates
+  + lease is preempted when another more suitable candidate is found
 - Lease pings are parallelized
 - Tests are included for third party strategies
+- Tests for disablement of the feature gate
+
+#### GA
+
+- Load test Coordinated Leader Election
+- Feature is enabled by default
 
 ### Upgrade / Downgrade Strategy
 
-If the `--leader-elect-resource-lock=coordinatedleases` flag is set and a
-component is downgraded from beta to alpha, it will need to either remove the
-flag or enable the alpha feature. All other upgrades and downgrades are safe.
+Upgrading requires enabling the feature gate `CoordinatedLeaderElection` and the group version `coordination.k8s.io/v1alpha2`. Downgrading will revert to the old leader election mechanism, but may have extra data in etcd for `LeaseCandidate` objects under the `coordination.k8s.io/v1alpha2` group version.
```

Comment:
now rereading the old sentence, it actually makes sense: when downgrading one cannot keep using the `--leader-elect-resource-lock=coordinatedleases` flag because it will error.